server/views/components/page-description/page-description.njk

```diff
@@ -18,7 +18,7 @@
     <div class="row">
       <div class="container">
         <div class="grid">
-          <div class="grid__cell grid__cell--shift-m1 {{ 'grid__cell--shift-l1' if model.lead }}">
+          <div class="grid__cell grid__cell--shift-m1 {{ 'grid__cell--shift-l1 grid__cell--shift-xl1' if model.lead }}">
```

Comment:
I think we've tried here, but the conditional – `if model.lead` – makes it tricky. I'll check.